docs/builds/guides/frameworks/react.md

```diff
@@ -67,18 +67,73 @@ The `<CKEditor>` component supports the following properties:
 * `editor` (required) &ndash; The {@link module:core/editor/editor~Editor `Editor`} constructor to use.
 * `data` &ndash; The initial data for the created editor. See the {@link builds/guides/integration/basic-api#interacting-with-the-editor Basic API} guide.
 * `config` &ndash; The editor configuration. See the {@link builds/guides/integration/configuration Configuration} guide.
-* `onInit` &ndash; A function called when the editor was initialized. It receives the initialized {@link module:core/editor/editor~Editor `editor`} as a parameter.
+* `id` &ndash; The editor ID. When that property changes the component restarts the editor with new data instead of setting it on an initialized editor.
+* `onReady` &ndash; A function called when the editor is ready with an {@link module:core/editor/editor~Editor `editor`} instance. This callback can be called after the initialization and after restarts of the editor.
 * `disabled` &ndash; A Boolean value. The {@link module:core/editor/editor~Editor `editor`} is being switched to read-only mode if the property is set to `true`.
 * `onChange` &ndash; A function called when the editor data has changed. See the {@link module:engine/model/document~Document#event:change:data `editor.model.document#change:data`} event.
 * `onBlur` &ndash; A function called when the editor was blurred. See the {@link module:engine/view/document~Document#event:blur `editor.editing.view.document#blur`} event.
 * `onFocus` &ndash; A function called when the editor was focused. See the {@link module:engine/view/document~Document#event:focus `editor.editing.view.document#focus`} event.
-* `onError` &ndash; A function called when the editor has crashed during the initialization. It receives the error object as a parameter.
+* `onError` &ndash; A function called when the editor has crashed during the initialization or during the runtime. It receives the error instance and the error details (`phase` and `willEditorRestart`) as arguments.
 
-The editor events callbacks (`onChange`, `onBlur`, `onFocus`) receive two parameters:
+The editor events callbacks (`onChange`, `onBlur`, `onFocus`) receive two arguments:
 
 1. An {@link module:utils/eventinfo~EventInfo `EventInfo`} object.
 2. An {@link module:core/editor/editor~Editor `Editor`} instance.
 
+### Context feature
+
+The `ckeditor5-react` integration comes with a ready-to-use component for the {@link features/context-and-collaboration-features Context feature} that is useful when used together with some {@link features/collaboration CKEditor 5 collaboration features}.
+
+```jsx
+import React, { Component } from 'react';
+import { CKEditor, CKEditorContext } from '@ckeditor/ckeditor5-react';
+
+// Assuming that application builds the editor from source.
+import Context from '@ckeditor/ckeditor5-core/src/context';
+import ClassicEditor from './ckeditor';
+
+class App extends Component {
```

Comment:
After fixing issues, the app builds with two not-looking-good editors:

![image](https://user-images.githubusercontent.com/2270764/94113066-9c336080-fe46-11ea-9c12-918df90d7fbe.png)

The `[data]` property for both editors is ignored (I guess because of lack of plugins).

Reply:
Yes, because of the lack of plugins. That's why I decided to use `./ckeditor` instead. I can add more info there though.

Reply:
Instead of creating a new file, we can add the definition of the editor in the component file. We will have all in one place.

Reply:
Do you mean creating a sample editor from source in a few lines?

Reply:
Yes, exactly the same as the editor that is exported by the file. 

```js
class CustomEditor /* ... extends */ { }
CustomEditor.builtinPlugins = [ /* ... */ ];
```